### DISCUSSION TITLE
Create IRC channel / Discord for this?

### DISCUSSION THREAD

**Discussion by Strykar** [Anime4000/RTL960x#244] [Ideas]
It might help foster a community and reduce common repetitive Issues / Discussions.

**Comment by Anime4000** [comment]
we do, it was created by other community member, not me 😅
https://discord.gg/JWdAzj9m4F

**Comment by Strykar** [comment]
I joined that Discord, I have also created the IRC channel named `##RTL960x` on the [Libera.chat](https://libera.chat/) network.
Would be nice to add both to the README once you approve of them?
